src/main/java/org/jitsi/videobridge/LipSyncHack.java

```diff
@@ -204,52 +284,40 @@ void onRTPTranslatorWillWriteAudio(
             if (acceptedVideoSSRCs.contains(receiveVideoSSRC)
                 || injections.containsKey(receiveVideoSSRC))
             {
-                // No need to hack this SSRC.
+                // No need to hack this SSRC, as it's either already accepted
```

Comment:
I have updated the explanatory comment. Please let me know if it still doesn't make sense.